### PR TITLE
feat: added info popup on create/edit and overview tab .

### LINKED ISSUE #6062: Virtual Events Video Tab: Add helper information on page
Please add helper information on the video tab page to guide users as follows. On the table heading and on the video edit page:
* Microlocations add an (i) icon. When the user scrolls over it show the info "You need to add rooms into the wizard first before they show up here."
* Video Source URL add an (i) icon. When the user scrolls over it show the info "This column shows the original link of the video. We do not recommend to share this link as users can access it without loggin into eventyay.com."
* "Join Video" add an (i) icon. When the user scrolls over it show the info "You can share the links in this columns with attendees of an event. On event or schedule schedule pages attendees of your event will see a Join Video button. These buttons will direct to the links shown here. Only organizers and registered ticket holders will be able to access these links."
* "Room Password" add an (i) icon. When the user scrolls over it show the info "The room password field can be used to communicate a password which is necessary to access online video rooms for example for external video services such a Zoom, Teams and Webex. The need for a password entry depends on the configuration of your video channel. The integrated Big Blue Button video solution in eventyay.com does not need a password as only ticket holders are able to access it. The difference between the password and PIN is that the password option is used for online access while PINs are used to access video rooms through the telephone."
* "Additional Information" add an (i) icon. When the user scrolls over it show the info "Additional Information fields can be used to share information such as phone access, PINs and other video room specific information."

### PR DESCRIPTION
**Pull request by nlok5923**
Fixes #6062 

#### Short description of what this resolves:
Added helper information 

#### Changes proposed in this pull request:

- added helper information on create/edit and overview tab. 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by Afzal-Ind — DISMISSED** [review]
LGTM!

File: app/components/forms/events/view/videoroom-form.hbs [hunk 1]
```diff
@@ -56,20 +56,32 @@
   {{/unless}}
   {{#if (not-eq this.data.stream.videoChannel.provider 'bbb')}}
     <div class="field">
-      <label class="required">{{t 'Video Source URL'}}</label>
+      <label class="required">{{t 'Video Source URL'}}
+      <span class="inline-display"> <UiPopup @content={{ "This column shows the original link of the video. We do not recommend to share this link as users can access it without loggin into eventyay.com."}} >
+            <i class="info icon"></i>
+            </UiPopup></span>
```
Comment (MeghalBisht) [inline app/components/forms/events/view/videoroom-form.hbs:62]: indentation

**Review by MeghalBisht — CHANGES_REQUESTED** [review]

File: app/styles/libs/_helpers.scss [hunk 2]
```diff
@@ -98,3 +98,7 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 .no-wrap {
   white-space: nowrap;
 }
+
+.inline-display {
+  display: inline-block;
```
Comment (MeghalBisht) [inline app/styles/libs/_helpers.scss:103]: there is already a class for inline-block please use that

File: app/components/forms/events/view/videoroom-form.hbs [hunk 3]
```diff
@@ -56,20 +56,32 @@
...
         @value={{this.data.stream.url}}
         @onChange={{action (mut this.data.stream.url)}} />
     </div>
     <div class="field">
-      <label>{{t 'Room Password'}}</label>
+      <label>{{t 'Room Password'}}
+      <span class="inline-display">
+             <UiPopup @content={{ "The room password field can be used to communicate a password which is necessary to access online video rooms for example for external video services such a Zoom, Teams and Webex. The need for a password entry depends on the configuration of your video channel. The integrated Big Blue Button video solution in eventyay.com does not need a password as only ticket holders are able to access it. The difference between the password and PIN is that the password option is used for online access while PINs are used to access video rooms through the telephone."}} >
+            <i class="info icon"></i>
+            </UiPopup> </span></label>
```
Comment (MeghalBisht) [inline app/components/forms/events/view/videoroom-form.hbs:74]: indentation

**Review by MeghalBisht — CHANGES_REQUESTED** [review]

File: app/components/forms/events/view/videoroom-form.hbs [hunk 4]
```diff
@@ -56,20 +56,32 @@
...
       <Widgets::Forms::LinkField
         @inputId="url"
         @value={{this.data.stream.url}}
         @onChange={{action (mut this.data.stream.url)}} />
     </div>
     <div class="field">
-      <label>{{t 'Room Password'}}</label>
+      <label>{{t 'Room Password'}}
+      <span class="inline-display">
+             <UiPopup @content={{ "The room password field can be used to communicate a password which is necessary to access online video rooms for example for external video services such a Zoom, Teams and Webex. The need for a password entry depends on the configuration of your video channel. The integrated Big Blue Button video solution in eventyay.com does not need a password as only ticket holders are able to access it. The difference between the password and PIN is that the password option is used for online access while PINs are used to access video rooms through the telephone."}} >
```
Comment (MeghalBisht) [inline app/components/forms/events/view/videoroom-form.hbs:72]: Also, please wrap in t

**Review by mariobehling — CHANGES_REQUESTED** [review]
Thank you. Could you change the following please:
* Please take out the effect of the tool tip and make it appear right away without any delay
* The way the icon appears currently stands out a lot. It looks larger and bold compared to the text. Therefore please make the icon to the same font size as the text and change the i-icon to an icon more similar to the following. Do not make it appear bolder than the text in front.
Screenshot from 2020-12-18 11-19-47

**Comment by mariobehling** [comment]
Also, please do not make the icon in front of the red * symbol. It should go behind it.

**Comment by nlok5923** [comment]
@mariobehling  sir shall i model the image provided by you as ui image for info icon as both class info icon and info-letter icon renders the icon "i".

**Comment by MeghalBisht** [comment]
shall i model the image provided by you as ui image for info icon as both class info icon and info-letter icon renders the icon "i".

use info circle

File: app/styles/libs/_helpers.scss [hunk 5]
```diff
@@ -98,3 +98,7 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 .no-wrap {
   white-space: nowrap;
 }
+
+.icon-color {
```
Comment (iamareebjamal) [inline app/styles/libs/_helpers.scss:102]: No need for this. Check how it is done in other places

File: app/templates/components/tables/default.hbs [hunk 6]
```diff
@@ -29,10 +29,55 @@
...
+            </div>
+            {{/if}}
+             {{#if (eq column.name "Video Source URL")}}
+             <div>
+              <span class="d-inline-block">{{column.name}}</span>
+              <span class="d-inline-block"> <UiPopup @content={{ "This column shows the original link of the video. We do not recommend to share this link as users can access it without loggin into eventyay.com."}}  @variation="basic">
+             <i class="info circle icon grey"></i>
+            </UiPopup></span>
+            </div>
+            {{/if}}
```
Comment (maze-runnar) [inline app/templates/components/tables/default.hbs:52]: please correct indentation throughout file.

File: app/components/forms/events/view/videoroom-form.hbs [hunk 7]
```diff
@@ -2,45 +2,51 @@
   <div class="ui grid">
     <div class="ui left aligned sixteen wide mobile eight wide computer column">
       <h3 class="ui header">
-        {{t 'Video Channel'}}
+        {{t "Video Channel"}}
```
Comment (iamareebjamal) [inline app/components/forms/events/view/videoroom-form.hbs:5]: Why making unnecessary changes?

Comment (nlok5923) [reply app/components/forms/events/view/videoroom-form.hbs:5]: Sir i used prettier for indentation which by default changes ' ->".

Comment (iamareebjamal) [reply app/components/forms/events/view/videoroom-form.hbs:5]: Revert it

File: app/templates/components/tables/default.hbs [hunk 8]
```diff
@@ -29,10 +29,62 @@
                 column.headerComponent
                 sorts=this.sorts column=column sortBy=this.sortBy sortDir=this.sortDir
               }}
-                {{column.name}}
+              {{column.name}}
               {{/component}}
             {{else}}
-              {{column.name}}
+            {{#if (eq column.name "Microlocation")}}
+            <div>
```
Comment (iamareebjamal) [inline app/templates/components/tables/default.hbs:36]: indentation

File: app/components/forms/events/view/videoroom-form.hbs [hunk 9]
```diff
@@ -56,23 +56,45 @@
   {{/unless}}
   {{#if (not-eq this.data.stream.videoChannel.provider 'bbb')}}
     <div class="field">
-      <label class="required">{{t 'Video Source URL'}}</label>
-      <Widgets::Forms::LinkField
-        @inputId="url"
-        @value={{this.data.stream.url}}
+      <span class="d-inline-block">
+        <label class="required">{{t 'Video Source URL'}}
+        </label>
```
Comment (iamareebjamal) [inline app/components/forms/events/view/videoroom-form.hbs:61]: Unnecessary change

Comment (nlok5923) [reply app/components/forms/events/view/videoroom-form.hbs:61]: Sir after doing proper indentation too those unnecessary changes go pushed.

Comment (iamareebjamal) [reply app/components/forms/events/view/videoroom-form.hbs:61]: No

**Comment by nlok5923** [comment]
@iamareebjamal  sir i had indented code correctly please review this pr.

**Comment by iamareebjamal** [comment]
Your indentation is still wrong and there are too many unrelated changes? Why adding span inline everywhere?
